public/js/main.js

```diff
@@ -15,6 +16,7 @@ import highlight from './highlight';
 if (document.readyState === 'interactive' || document.readyState === 'complete') {
```

Comment:
We could use o-autoinit here instead of our own bespoke code - https://github.com/Financial-Times/o-autoinit